src/core/binding.c

```diff
@@ -1058,10 +1058,13 @@ QuicBindingProcessStatelessOperation(
         goto Exit;
     }
 
+    CXPLAT_ROUTE Route;
+    Route.LocalAddress = RecvDatagram->Tuple->LocalAddress;
+    Route.RemoteAddress = RecvDatagram->Tuple->RemoteAddress;
```

Comment:
Why aren't we replacing the `Tuple` in the `RecvDatagram` with `Route`? Then that could have simply been passed right back.

Reply:
Yeah that's the plan, but it's a big change by itself and I'd like to put it in its own PR.